sdk/monitor/monitor-opentelemetry/samples-dev/src/01-basic-connection.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * Basic Connection Example
+ *
+ * This example demonstrates how to configure Azure Monitor OpenTelemetry
+ * using the APPLICATIONINSIGHTS_CONNECTION_STRING environment variable.
+ */
+
+export class BasicConnectionExample {
+  static async run() {
+    const { useAzureMonitor } = await import("@azure/monitor-opentelemetry");
+
+    try {
+      // Configure Azure Monitor with connection string from environment
+      const options = {
+        azureMonitorExporterOptions: {
+          connectionString:
+            process.env.APPLICATIONINSIGHTS_CONNECTION_STRING || "<your connection string>",
+        },
+      };
+
+      // Enable Azure Monitor integration using the useAzureMonitor function
+      useAzureMonitor(options);
+
+      console.log("Azure Monitor configured with connection string from environment variable");
+      console.log("Connection string source: APPLICATIONINSIGHTS_CONNECTION_STRING");
+      console.log("Telemetry will be sent to Azure Application Insights");
+
+      // Simulate some application work
+      console.log("Simulating application work...");
+
+      // This would be your actual application logic
+      // The telemetry data will be automatically collected and sent to Azure Monitor
+
+      await new Promise((resolve) => setTimeout(resolve, 1000));
+      console.log("Application work completed");
+    } catch (error) {
+      console.error("Error configuring Azure Monitor:", error);
+    }
+  }
+}
+
+// Usage instructions
+if (require.main === module) {
```

Comment:
Have you considered ESM as well as CJS here? 

I wonder if we should have a setup that works for both. It'll require having the instrumentation set-up file separate from the main app startup and using either `--require` or `--loader` or something like that. I think `--require` works.

Then, you could showcase what most customers commonly get confused over - how do I enable telemetry in ESM? 

See the [OTel docs](https://github.com/open-telemetry/opentelemetry-js/blob/main/doc/esm-support.md) for more information or feel free to reach out to me if you want to dig further into it.